test/OpenTelemetry.Tests/Impl/Trace/SpanTest.cs

```diff
@@ -807,7 +803,6 @@ public async Task EndSpan_Properties()
             span.AddEvent(EventDescription, attributes);
 
             var link = new Link(contextLink);
-            span.AddLink(link);
             span.Status = Status.Cancelled;
 
             var spanEndTime = PreciseTimestamp.GetUtcNow();
```

Comment:
The assert `Assert.Single(span.Links);` will fail not. But since `StartRootSpan` doesn't take _parent links_, is it even possible to have Links on a root span?

Reply:
> since StartRootSpan doesn't take parent links, is it even possible to have Links on a root span

it has links https://github.com/open-telemetry/opentelemetry-dotnet/blob/master/src/OpenTelemetry.Abstractions/Trace/ITracer.cs#L61

it's fine for root span to have links - it just doesn't have parent.